var/spack/repos/builtin/packages/nest/package.py

```diff
@@ -107,7 +110,7 @@ def cmake_args(self):
 
         if '+python' in self.spec:
             version = self.spec['python'].version[0]
-            args.append('-Dwith-python={0}'.format(version))
+            args.append('-Dwith-python=ON')
```

Comment:
The line:
```
version = self.spec['python'].version[0]
```
needs to be removed too, since `version` is assigned but never used.

Reply:
Done.